src/docs/tasks.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Tasks
+---
+
+# Tasks
+
+Eclipse Theia users can execute tasks in the context of a workspace to automate certain steps in their workflow.
```

Comment:
There is also support for `user`-level tasks (https://github.com/eclipse-theia/theia/issues/7446) but we don't necessarily need to mention them at this time.

Reply:
Thanks, good input! I removed "in the context of a workspace" as it is vague anyway and added a sentence at the end that tasks can be specified in the workspace or on user level.